crypto/x509v3/v3_purp.c

```diff
@@ -765,7 +765,8 @@ static int no_check(const X509_PURPOSE *xp, const X509 *x, int ca)
  * These are:
  * 1. Check issuer_name(subject) == subject_name(issuer)
  * 2. If akid(subject) exists check it matches issuer
- * 3. If key_usage(issuer) exists check it supports certificate signing
+ * 3. Check that signing algorithm matches signature algorithm
+ * 4. If key_usage(issuer) exists check it supports certificate signing
```

Comment:
"If ... exists check it supports ..." -> "If ... exists, check whether/if/that it supports..."   (note the added ",").

Applies also to  2.

Reply:
Will fix